docs/walkthrough/index.md

```diff
@@ -0,0 +1,255 @@
+  --output tsv
+```
+
+## Create an AKS Cluster
+
+(From the [AKS Cluster Quickstart](https://docs.microsoft.com/en-us/azure/aks/kubernetes-walkthrough#create-aks-cluster))
+
+```bash
+az aks create \
+  --name PromitorCluster \
+  --resource-group PromitorRG \
+  --node-count 1 \
+  --generate-ssh-keys
+```	
+
+If the Kubernetes command line tool `kubectl` isn't already installed, you can install it with 
+
+```bash
+az aks install-cli
+```
+
+Then get your cluster's credentials with
+
+```bash
+az aks get-credentials \
+  --name PromitorCluster \
+  --resource-group PromitorRG
+```
+
+Verify your credentials and check that your cluster is up and running with `kubectl get nodes`.
+
+
+## Install Helm and Tiller
+
+[Install Helm](https://helm.sh/docs/using_helm/#installing-helm) on your local machine if you don't already have it.
+
+You'll use Helm to install Tiller, the server-side component of Helm. For clusters with RBAC (Role-Based Access Control), you'll need to set up a service account for Tiller:
+- Create a file called `helm-rbac.yaml` with the following:
+
+```YAML
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: tiller
+  namespace: kube-system
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRoleBinding
+metadata:
+  name: tiller
+roleRef:
+  apiGroup: rbac.authorization.k8s.io
+  kind: ClusterRole
+  name: cluster-admin
+subjects:
+  - kind: ServiceAccount
+    name: tiller
+    namespace: kube-system
+```
+
+- Create the service account and role binding specified in the above file with the `kubectl apply` command:
+
+```bash
+kubectl apply -f helm-rbac.yaml
+```
+
+- To deploy Tiller into your AKS cluster, use the `helm init` command:
+
+```bash
+helm init --service-account tiller
+```
+
+## Deploy Promitor to your cluster using Helm
+
+In addition to the helm chart, you'll need a values file with secrets & a metric declaration file (these can also be the same file for ease of use). The yaml file below will scrape one metric, queue length, from the queue created above.
+
+```yaml
+azureAuthentication:
+  appId: <guid-sp-app-id>
+  appKey: <guid-sp-generated-password>
+
+azureMetadata:
+  tenantId: <guid-tenant-id>
+  subscriptionId: <guid-subscription-id>
+  resourceGroupName: PromitorRG
+metricDefaults:
+  aggregation:
+    interval: 00:05:00
+  scraping:
+    schedule: "* * * * *"
+metrics:
+  - name: demo_queue_size
+    description: "Amount of active messages of the 'demo_queue' queue"
+    resourceType: ServiceBusQueue
+    namespace: <service-bus-namespace>
+    queueName: demo_queue
+    azureMetricConfiguration:
+      metricName: ActiveMessages
+      aggregation:
+        type: Total
+```
+
+With this file created, you should be able to deploy Promitor with `helm install`. You'll need to be in the top level directory of the Promitor repository for this command to run, or you'll need to edit the chart location.
+
+```bash
+helm install ./charts/promitor-agent-scraper \
```

Comment:
See https://github.com/tomkerkhove/promitor/pull/539 - We now have a public Helm chart.

Can you use this one please?